requirements.txt

```diff
@@ -1,5 +1,5 @@
 pytest~=2.8
-requests_oauthlib~=0.8.0
+requests_oauthlib>=0.8.0<1.1
```

Comment:
Thanks, I just gave this a shot and I can confirm everything seems to be OK with requests 1.0.0. I think the major version number bump was from dropping older Python support, but we already did that in python-asana, so this looks pretty straightforward. I'm going to pull in these changes and tweak the versions to be `<2.0`, since in theory all future versions of `1.*` for requests_oauthlib should be backwards compatible up to version 2. Thanks!